htdocs/compta/paymentbybanktransfer/index.php

```diff
@@ -125,8 +135,9 @@
 if ($socid) {
 	$sql .= " AND f.fk_soc = ".((int) $socid);
 }
-
+$sqlForSalary = "SELECT * FROM llx_salary as s, llx_prelevement_demande as pd WHERE s.rowid = pd.fk_salary AND s.paye = 0 AND pd.traite = 0";
```

Comment:
SELECT * 
is forbidden in Dolibarr. Each fields must be explicitly named.
It makes not possible to do impact analysis on code when we rename a column for example.

Reply:
@lamrani002 Still this point to fix.